packages/legend-query-builder/src/components/__tests__/QueryBuilderPostFilterPanel.test.tsx

```diff
@@ -109,16 +114,12 @@ test(
         ),
       );
     });
-    const queryBuilderFilterPanel = await waitFor(() =>
```

Comment:
I replaced all the instances of `await waitFor` with `await findBy` in this test file because it's cleaner, so that's why there's so many changes here.